### PR TITLE
Automatic update of Microsoft.Extensions.DependencyInjection.Abstractions to 2.1.1

### PR DESCRIPTION
**Pull request by NuKeeperBot**
NuKeeper has generated a minor update of `Microsoft.Extensions.DependencyInjection.Abstractions` to `2.1.1` from `2.0.0`
`Microsoft.Extensions.DependencyInjection.Abstractions 2.1.1` was published at `2018-06-18T20:17:38Z`, 5 months ago

1 project update:
Updated `src/JustEat.StatsD/JustEat.StatsD.csproj` to `Microsoft.Extensions.DependencyInjection.Abstractions` `2.1.1` from `2.0.0`

[Microsoft.Extensions.DependencyInjection.Abstractions 2.1.1 on NuGet.org](https://www.nuget.org/packages/Microsoft.Extensions.DependencyInjection.Abstractions/2.1.1)

This is an automated update. Merge only if it passes tests
**NuKeeper**: https://github.com/NuKeeperDotNet/NuKeeper

### CODE REVIEW

File: src/JustEat.StatsD/JustEat.StatsD.csproj [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' != 'net451' ">
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.1.1" />
```
Comment (martincostello) [inline src/JustEat.StatsD/JustEat.StatsD.csproj]: We don't want this because it's being pinned low on purpose, right?

Comment (AnthonySteele) [reply src/JustEat.StatsD/JustEat.StatsD.csproj]: Ah right, I thought it was only a test dependency. I will exclude it from the filter and close this PR.

**Comment by AnthonySteele** [comment]
I am pointing a NuKeeepr cron job at this public repo.
The package filter is expected to be `--include (Test|xunit|Moq|Shouldly|BenchmarkDotNet)`

File: src/JustEat.StatsD/JustEat.StatsD.csproj (hunk 1 above)
Comment (martincostello) [reply src/JustEat.StatsD/JustEat.StatsD.csproj]: Also, is there something slightly misconfigured? The bot seems to have a dual identity:

image

**Pull request closed without merging**